pkgs/development/mobile/androidenv/emulate-app.nix

```diff
@@ -1,16 +1,19 @@
 { composeAndroidPackages, stdenv, lib, runtimeShell }:
-{ name, app ? null
+{ name
+, app ? null
 , platformVersion ? "33"
 , abiVersion ? "armeabi-v7a"
 , systemImageType ? "default"
-, enableGPU ? false
-, extraAVDFiles ? []
+, enableGPU ? false # Enable GPU acceleration. It's deprecated, instead use `configOptions` below.
```

Comment:
It should use a warning, something similar to what we did in cmake:

https://github.com/NixOS/nixpkgs/blob/e274bc3056e1062f9b98d8f473ae8b65ffe8c6b5/pkgs/by-name/cm/cmake/package.nix#L20C21-L20C21

Reply:
Thanks for the review and the hint. Fixed in [51dd776](https://github.com/NixOS/nixpkgs/pull/265504/commits/51dd776c0cff30f3056a1354383b9dc2c6208e34)